R/stations.R

```diff
@@ -149,10 +149,10 @@ stations_dl_internal <- function(skip = NULL, verbose = FALSE, quiet = FALSE,
 
   # Get normals data
   normals <- stations_normals()
-
+  
   if(verbose) message("Trying to access stations data frame")
-  resp <- httr::GET(getOption("weathercan.urls.stations"))
 
+  resp <- get_check(getOption("weathercan.urls.stations"))
   if(httr::http_error(resp)) {
     stop("Cannot reach ECCC stations list, please try again later (",
          getOption("weathercan.urls.stations"), ")", call. = FALSE)
```

Comment:
Definitely agree, this could be changed as follows:

```suggestion
  resp <- get_check(getOption("weathercan.urls.stations"), task = "access stations list")
```